script/hassfest/translations.py

```diff
@@ -132,12 +132,22 @@ def translation_value_validator(value: Any) -> str:
     - prevents combined translations
     """
     value = cv.string_with_no_html(value)
-    value = cv.string_no_single_quoted_placeholders(value)
+    value = string_no_single_quoted_placeholders(value)
     if RE_COMBINED_REFERENCE.search(value):
         raise vol.Invalid("the string should not contain combined translations")
     return str(value)
 
 
+def string_no_single_quoted_placeholders(value: str) -> str:
+    """Validate that the value does not contain placeholders inside single quotes."""
+    regex = re.compile(r"'{\w+}'")
```

Comment:
May be we can define the regex once for perfomance reasons?

Reply:
I continue to blame the pattern in `string_with_no_html`, but `translations.py` has a better pattern for regex.

Fixed!

Reply:
Thnx!